tests/polynomial_test.py

```diff
@@ -59,9 +59,9 @@ def args_maker():
         [jnp.zeros(leading, p.dtype), p, jnp.zeros(trailing, p.dtype)]),
 
     # order may differ (jnp.sort doesn't deal with complex numbers)
```

Comment:
I think this comment is no longer true... we could replace `np.sort` with `jnp.sort` below, though that's not entirely necessary.

Reply:
Indeed! I improved it.